static/src/questionnaire-detail.js

```diff
@@ -0,0 +1,50 @@
+import Vue from 'vue/dist/vue.js'
+import Dropzone from 'dropzone';
+import Question from './details/Question'
+import Answer from './details/Answer'
+
+
+var url = "/api/question/";
+
+var question_app = new Vue({
+  el: '#question-detail-app',
+  data: {
+    results: {},
+  },
+  methods: {
+    fetchQuestionData: function fetchQuestionData() {
+      var _this = this;
+
+      axios.get(url).then(function (response) {
+        _this.results = response.data;
+        _this.saveResponseFiles(response.data);
+      });
+    },
+    saveResponseFiles: function saveResponseFiles(question_data) {
+      for (var question_id in question_data) {
```

Comment:
Ne pas utiliser `for ... in` pour itérer sur les arrays en JS, c'est fait pour les objects, et du coup ca peut faire des trucs bizarres : https://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-a-bad-idea

A la place le truc stylé c'est un map : 
```
question_data.map((data) => { 
   var response_files = data.response_files;
...
 })
```

Reply:
C'est de l'or ce conseil, merci  :+1: